### ISSUE TITLE
Investigate integration of GA4GH APIs with Excel

### ISSUE THREAD

**Issue by diekhans** [BD2KGenomics/ga4gh-integration#25]
Excel is a commonly used for bioinformatics analysis.  This approach appears to be especially popular with research coming from a biological background without a formal bioinformatics background  (who often use R).

Documentation, examples, libraries, on how to integrate GA4GH with excel might be attract an important class of users who would not otherwise use an API.

Of course, there is nothing novel  Excel/ReST integration:
http://www.finfolio.com/blog/using-rest-api-data-in-microsoft-excel

**Comment by david4096** [comment]
Yes, I agree! If bioinformatics is anything like physics, PIs still love Excel! Being able to import the BRCA exchange data into excel using the GA4GH API would be a very motivating use case.
